cmake/common.cmake

```diff
@@ -146,95 +134,27 @@ include_directories(${PROJECT_SOURCE_DIR}
                     ${TARGET_FS}/usr/include/
                    )
 
-if ("${TARGET_CPU}" STREQUAL "A72" OR "${TARGET_CPU}" STREQUAL "A53")
-    set(SYSTEM_LINK_LIBS
-        tivision_apps
-        edgeai-tiovx-kernels
-        edgeai-apps-utils
-        m
-        yaml-cpp
-        )
-else()
-    set(SYSTEM_LINK_LIBS
-        edgeai-tiovx-kernels
-        edgeai-apps-utils
-        m
-        yaml-cpp
-        app_utils_init
-        libvx_conformance_engine.a
-        vx_conformance_tests
-        vx_conformance_tests_testmodule
-        vx_framework
-        vx_kernels_host_utils
-        vx_kernels_openvx_core
-        vx_kernels_openvx_ext
-        vx_kernels_openvx_ext_tests
-        vx_kernels_target_utils
-        vx_kernels_test_kernels
-        vx_kernels_test_kernels_tests
-        vx_platform_pc
-        vx_target_kernels_dsp
-        vx_target_kernels_ivision_common
-        vx_target_kernels_openvx_core
-        vx_target_kernels_openvx_ext
-        vx_target_kernels_source_sink
-        vx_target_kernels_tutorial
-        vx_tiovx_internal_tests
-        vx_tiovx_tests
-        vx_tutorial
-        vx_utils
-        vx_vxu
-        algframework_x86_64
-        c6xsim_x86_64_C66
-        dmautils_x86_64
-        vxlib_bamplugin_x86_64
-        vxlib_x86_64
-        app_utils_mem
-        app_utils_hwa
-        app_utils_iss
-        ti_imaging_aealg
-        ti_imaging_dcc
-        vx_kernels_hwa
-        vx_kernels_hwa_tests
-        vx_kernels_imaging
-        vx_kernels_imaging_tests
-        vx_target_kernels_dmpac_dof
-        vx_target_kernels_dmpac_sde
-        vx_target_kernels_imaging_aewb
-        vx_target_kernels_j7_arm
-        vx_target_kernels_vpac_ldc
-        vx_target_kernels_vpac_msc
-        vx_target_kernels_vpac_nf
-        vx_target_kernels_vpac_viss
-        ti_imaging_awbalg
-        bl_filter_lib
-        cac
-        DOF
-        ee
-        flexcc
-        flexcfa_vpac3
-        glbce
-        h3a
-        ldc
-        nsf4
-        nsf4_wb
-        rawfe
-        RawHistogram
-        scalar
-        sde_hw
-        utils
-        VXLIB_triangulatePoints_i32f_o32f_lib_x86_64
-        m
-       )
-endif()
+set(SYSTEM_LINK_LIBS
+    tivision_apps
+    edgeai-tiovx-kernels
+    edgeai-apps-utils
+    m
+    yaml-cpp
+    )
 
 if ("${TARGET_OS}" STREQUAL "LINUX")
+if ("${TARGET_CPU}" STREQUAL "A72" AND "${TARGET_CPU}" STREQUAL "A53")
```

Comment:
@ravi-rahul , It should be ${TARGET_CPU}" STREQUAL "A72" **OR** "${TARGET_CPU}" STREQUAL "A53"

Reply:
Thanks for finding this @sinha-shreyash 
Fixed this now